tao_compiler/mlir/disc/IR/hlo_disc_ops.cc

```diff
@@ -194,6 +194,12 @@ LogicalResult QuantizedDynamicConvOp::verify() {
   return CommonVerifyForQuantizedComputeIntensiveOp(this);
 }
 
+//===----------------------------------------------------------------------===//
+// SparseReshapeOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult SparseReshapeOp::verify() { return Verify(*this); }
```

Comment:
better to add a real verifier, instead of skipping it.

Reply:
done